packages/loopring_v3/contracts/impl/BasicDepositContract.sol

```diff
@@ -83,19 +86,15 @@ contract BasicDepositContract is IDepositContract, ReentrancyGuard
     function deposit(
         address from,
         address token,
-        uint amount
+        uint amount,
+        bytes calldata /*auxiliaryData*/
```

Comment:
it is ok to delete `nonReentrant` as it's only callable by the exchange, and all external functions in ExchangeV2 are `nonReentrant`.

Reply:
Removed `nonReentrant` just to save some gas for deposits.